### DISCUSSION TITLE
从节点不工作, 没有消费消息

### DISCUSSION THREAD

**Discussion by fade-away** [apache/rocketmq#6913] [Q&A]
![image](https://github.com/apache/rocketmq/assets/8369754/1fdd5bbd-53b1-4f6b-8e96-081d32b95ad9)

版本 4.9.0
搭建方式： 双主双从
![企业微信截图_16868778243746](https://github.com/apache/rocketmq/assets/8369754/0f84920f-31b5-4704-a820-d7e9073b0610)

**Comment by RongtongJin** [comment]
看一下配置信息和store.log

**Comment by mxsm** [comment]
首先你对比一下配置和官网给的有没有什么区别，然后再检查一下Slave Broker中是否有同步到Master Broker的数据，数据文件中是否有数据。如果没有就是配置的问题。
